cloud/pkg/common/util/util.go

```diff
@@ -0,0 +1,55 @@
+/*
+Copyright 2023 The KubeEdge Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	metav1 "k8s.io/api/core/v1"
+
+	"github.com/kubeedge/kubeedge/common/constants"
+)
+
+// IsEdgeNode checks whether a node is an Edge Node
+// only if label {"node-role.kubernetes.io/edge": ""} exists, it is an edge node
+func IsEdgeNode(node *metav1.Node) bool {
```

Comment:
Maybe we already have the func `IsEdgeNode`?

Reply:
> Maybe we already have the func `IsEdgeNode`?

Except in the conformance, there is no func `IsEdgeNode` before. But @ZhengXinwei-F will also achieve `validateNode` in the upgrade framework, this func `IsEdgeNode` will be removed when imageprepull used the framework.